src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -38,19 +38,17 @@ class DebugCommand extends Command
     private $projectDir;
     private $bundlesMetadata;
     private $twigDefaultPath;
-    private $rootDir;
     private $filesystemLoaders;
     private $fileLinkFormatter;
 
-    public function __construct(Environment $twig, string $projectDir = null, array $bundlesMetadata = [], string $twigDefaultPath = null, string $rootDir = null, FileLinkFormatter $fileLinkFormatter = null)
+    public function __construct(Environment $twig, string $projectDir = null, array $bundlesMetadata = [], string $twigDefaultPath = null, FileLinkFormatter $fileLinkFormatter = null)
```

Comment:
we need an upgrade path before changing the order or arguments

Reply:
Opps, sure I missed that

Reply:
Done in https://github.com/symfony/symfony/pull/31688